### PR TITLE
Liquidity Pool Section Update (#1)

### PR DESCRIPTION
**Pull request by tenuki**
AMM pool section update

For easier preview of resulting document use this link: https://github.com/alexgo-io/alexlab-doc/blob/680ea52e527b7930c8a28c739f1c35553aadb541/product-features/liquidity-pool/README.md

### CODE REVIEW

File: product-features/liquidity-pool/README.md [hunk 1]
```diff
@@ -1,17 +1,60 @@
 # 🐋 Liquidity Pool
 
-## Liquidity Pool
+## Getting Started
 
-You can become a liquidity provider by submitting two tokens to the liquidity pool to begin earning a share of the trading fees.
```
Comment (Hadan-ALEX) [inline product-features/liquidity-pool/README.md:5]: You can become a liquidity provider on ALEX by submitting two tokens to the liquidity pool to begin earning a share of the pool trading fees.

File: product-features/liquidity-pool/README.md [hunk 2]
```diff
@@ -1,17 +1,60 @@
...
+### What are Liquidity Pools?
 
-The trading fee rebate is 0.3% per transaction of which 50% goes to Liquidity Providers and 50% goes to the platform.
+Liquidity pools are crowdfunded collections of crypto assets held in a smart contract, designed to provide liquidity for decentralized exchanges (DEXs) and support various decentralized finance (DeFi) protocols.
 
-_Example:_
+While their applications are multiple, ranging from lending and borrowing platforms to algorithmic protocols for stablecoins, their primary use is on DEXs. In this case, liquidity pools enable users to trade crypto assets without the need for a centralized intermediary, serving as reserves of assets that users can trade against.
```
Comment (Hadan-ALEX) [inline product-features/liquidity-pool/README.md:9]: please replace "multiple" with "diverse"

**Review by Hadan-ALEX — COMMENTED** [review]
Very well written! Just one comment made on line 9, please replace "multiple" with "diverse." Thank you.

File: product-features/liquidity-pool/README.md (hunk 2 above)
Comment (sofinico) [reply product-features/liquidity-pool/README.md]: Done ✅

**Pull request merged**